test/src/test/resources/fixtures/field removed/top-level-field-is-removed-for-a-subscription-with-namespaced-field.yml

```diff
@@ -0,0 +1,60 @@
+name: "top level field is removed for a subscription with namespaced field"
+enabled: true
+# language=GraphQL
+overallSchema:
+  CommentService: |
+    directive @toBeDeleted on FIELD_DEFINITION
+    type Query {
+      commentById(id: ID): Comment @toBeDeleted
+    }
+    type Subscription {
+      commentsApi: CommentsApi @namespaced
+    }
+    type CommentsApi {
+      onCommentUpdated(id: ID): Comment @toBeDeleted
+    }
+    type Comment {
+      id: ID
+    }
+# language=GraphQL
+underlyingSchema:
+  CommentService: |
+    type Query {
+      commentById(id: ID): Comment
+    }
+    type Subscription {
+      commentsApi: CommentsApi
+    }
+    type CommentsApi {
+      onCommentUpdated(id: ID): Comment
+    }
+    type Comment {
+      id: ID
+    }
+# language=GraphQL
+query: |
+  subscription {
+    commentsApi {
+      onCommentUpdated(id: "C1") {
+        id
+      }
+    }
+  }
+variables: { }
+serviceCalls: [ ]
+# language=JSON
+response: |-
+  {
+    "errors": [
+      {
+        "locations": [],
+        "message": "null",
+        "extensions": {
+          "classification": "ValidationError"
```

Comment:
So this is an error from the introspection service? Why is the error message null?

Reply:
nah, this error is coming from the transform, look at this comment https://github.com/atlassian-labs/nadel/pull/649/files/5774cd1580b7dcaeaecb6b6b7d46bdea52b078eb#r1870548215

Reply:
Ah I see